frontend/src/pages/GraphPF/components/groupExpanded.tsx

```diff
@@ -242,6 +255,24 @@ const BaseGroupExpandedComponent: React.FunctionComponent<BaseGroupExpandedProps
     >
       <Layer id={GROUPS_LAYER}>
         <g ref={refs} onContextMenu={onContextMenu} onClick={onSelect} className={innerGroupClassName}>
+          {!!isFocused && (
```

Comment:
```suggestion
          {isFocused && (
```

I learned that the `!!` is unnecessary when the context is already a conditional.